Tools/ardupilotwaf/boards.py

```diff
@@ -448,7 +448,41 @@ def configure_env(self, cfg, env):
                 CANARD_ENABLE_DEADLINE = 1,
             )
 
+        if cfg.options.trusted_flight_issuer and cfg.options.trusted_flight_root_certificate:
+            # prepare a temp file to embed issuer string into ROMFS
+            with open('/tmp/trusted_flight_issuer.tmp', 'w') as f:
```

Comment:
should use the python tmpfile capability, both for security and cleanup

Reply:
or better putting the file in the build tree? Best not to create files outside of buildroot

Reply:
Can't use tempfile because it's cleaned up as soon as fd is closed. Moved it to buildroot instead